src/NHSD.BuyingCatalogue.SolutionLists.Persistence/Repositories/SolutionListRepository.cs

```diff
@@ -29,13 +30,13 @@ public sealed class SolutionListRepository : ISolutionListRepository
                            ON sol.Id = sc.SolutionId
                    INNER JOIN dbo.Capability AS cap
                            ON cap.Id = sc.CapabilityId
-              LEFT OUTER JOIN dbo.FrameworkSolutions AS fs
+                   INNER JOIN dbo.FrameworkSolutions AS fs
                            ON sol.Id = fs.SolutionId 
                             
```

Comment:
Can you remove the blank line here? I don't think we need it.

Reply:
Removed Blank line in query